### DISCUSSION TITLE
Top three suggestions to improve the docs?

### DISCUSSION THREAD

**Discussion by bruth** [nats-io/nats.docs#572] [Q&A]
Of course, you are welcome to provide more! Suggestions can include anything:

- Layout and style
- Search (we all know search needs improving..)
- Consistency of the content/prose 
- Organization of the navigation
- Missing topics
- Particular guides that would be valuable
- Reference information
- etc...

**Comment by ioanluca** [comment]
Hi, a guide that could be valuable is how to achieve Kafka style consumer group load balancing based on partitions (with automatic assignment). I know NATS can already help with that by using [subject mapping](https://docs.nats.io/nats-concepts/subject_mapping#deterministic-subject-token-partitioning) but it'd be interesting to know how partition rebalancing could be achieved. Thanks!

**Reply by bruth** [reply]
Thanks @ioanluca! This has been an increasingly common request.

**Reply by ioanluca** [reply]
Hi! Are there any news on this? What's the best place to track the progress for these kind of feature requests? Thanks!
